vendor/github.com/vmware/go-vcloud-director/v2/govcd/vapp.go

```diff
@@ -1128,9 +1128,22 @@ func (vapp *VApp) UpdateOrgNetworkAsync(networkSettingsToUpdate *VappNetworkSett
 	fenceMode := types.FenceModeBridged
 	if isFenced {
 		fenceMode = types.FenceModeNAT
-		networkToUpdate.Configuration.Features = &types.NetworkFeatures{
-			FirewallService: &types.FirewallService{IsEnabled: *networkSettingsToUpdate.FirewallEnabled},
-			NatService:      &types.NatService{IsEnabled: *networkSettingsToUpdate.NatEnabled, NatType: "ipTranslation", Policy: "allowTrafficIn"}}
+		if networkToUpdate.Configuration.Features != nil {
```

Comment:
Please add a comment about the purpose of the below if blocks.

Reply:
added in govcd PR

Reply:
Thanks, could you just move it one line below the `} else {`

<img width="1058" alt="Screenshot 2020-06-23 at 14 08 25" src="https://user-images.githubusercontent.com/11330689/85396854-2a9ff980-b55b-11ea-97ba-abfe0221be9b.png">

Reply:
done